api/main.go

```diff
@@ -102,8 +102,8 @@ func main() {
 		scalerTicker := scalerDispatcher.RunEvery(time.Minute * 5)
 		defer scalerTicker.Stop()
 
-		// todo: get job expiry from config env var
-		jobJanitor := job.NewJanitor(jobStore, time.Hour*24)
+		expiry := cfg.JobExpiryHour()
```

Comment:
We should always deal with `time.Duration` types when dealing with time, never integers. Integers add an unnecessary level of indirection. This should be `cfg.JobExpiry()`, and it should return a `time.Duration`.